src/pages/{StoreCollection.slug}.tsx

```diff
@@ -73,10 +73,12 @@ function Page(props: Props) {
       */}
 
       <div className="product-listing__breadcrumb / grid-content">
```

Comment:
I think we can remove this `div` here and add those classes to the `section` below.

Reply:
Thank you for the suggestion, i will apply